### PR TITLE
Updating templates, styles, and graphql query to grab social media

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Adds social media output to facility pages.

## Definition of done
- [ ] Social media appears on pages where users have entered values

### CODE REVIEW

File: src/platform/site-wide/sass/merger.scss [hunk 1]
```diff
@@ -607,6 +607,10 @@ img[data-srcset] {
   position: relative;
 }
 
+.feature-grey {
+  background-color: #f1f1f1;
+}
+
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/merger.scss]: We actually have utilities classes for [colors](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/color.html), to save on some color rules. For this shade of gray, we can use the class `.vads-u-color--gray-lightest`.

File: src/site/layouts/health_care_local_facility_page.drupal.liquid [hunk 2]
```diff
@@ -165,6 +165,49 @@
...
+                                        </div>
+                                    {% endif %}
+                                </div>
+
+                                <div class="usa-grid usa-grid-full">
+                                    {% if fieldEmailSubscription != empty %}
+                                        <div class="social-links social-links-email usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldEmailSubscription.url.path }}"><i class="fas fa-envelope"></i> {{ fieldEmailSubscription.title }}</a>
+                                        </div>
+                                    {% endif %}
```
Comment (ncksllvn) [inline src/site/layouts/health_care_local_facility_page.drupal.liquid]: Should the if-statement wrap the `<div class="usa-grid usa-grid-full">` as well?

**Review by ncksllvn — COMMENTED** [review]
Looks great! Just a couple small comments

File: src/site/layouts/health_care_local_facility_page.drupal.liquid (hunk 2 above)
Comment (ethanteague) [reply src/site/layouts/health_care_local_facility_page.drupal.liquid]: Yeah, you are right - I was just being lazy, @ncksllvn

File: src/platform/site-wide/sass/merger.scss (hunk 1 above)
Comment (ethanteague) [reply src/platform/site-wide/sass/merger.scss]: Sweet!

Comment (ethanteague) [reply src/platform/site-wide/sass/merger.scss]: Actually, that class appears to target font color. I did a search for `gray-lightest` and found the color variable, and inserted it for .feature-grey, @ncksllvn

Comment (ncksllvn) [reply src/platform/site-wide/sass/merger.scss]: Ah, sorry! Looks like there are classes for [background color](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/background-color.html) as well. Sorry - I'm new to our design system upgrade as well!

**Review by ncksllvn — APPROVED** [review]
LGTM 👍

**Pull request merged**